src/firefly/java/edu/caltech/ipac/firefly/server/dpanalyze/SofiaAnalyzer.java

```diff
@@ -0,0 +1,285 @@
+package edu.caltech.ipac.firefly.server.dpanalyze;
+
+import edu.caltech.ipac.firefly.core.FileAnalysisReport;
+import edu.caltech.ipac.firefly.data.sofia.Sofia1DSpectraExtractor;
+import edu.caltech.ipac.firefly.data.sofia.SofiaSpectraModel;
+import edu.caltech.ipac.firefly.data.sofia.VOSpectraModel;
+import edu.caltech.ipac.firefly.server.ServerContext;
+import edu.caltech.ipac.table.DataGroup;
+import edu.caltech.ipac.table.io.VoTableReader;
+import edu.caltech.ipac.table.io.VoTableWriter;
+import edu.caltech.ipac.util.download.FailedRequestException;
+import edu.caltech.ipac.util.download.URLDownload;
+import nom.tam.fits.Header;
+
+import java.io.File;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.*;
+
+import static edu.caltech.ipac.table.TableUtil.Format.VO_TABLE_TABLEDATA;
+
+/*
+ Analyzer for SOFIA extracted Spectra from FITS 'image'
+ */
+@DataProductAnalyzerImpl(id = "analyze-sofia")
+public class SofiaAnalyzer implements DataProductAnalyzer {
+
+
+    @Override
+    public FileAnalysisReport analyze(FileAnalysisReport inputReport, File inFile, String analyzerId, Map<String, String> params) {
+        return inputReport;
+    }
```

Comment:
Unnecessary override.  Same as default impl.

Reply:
I missed this but java now allows interfaces to have default implementations. `DataProductAnalyzer` was the first time I ever wrote an interface like this. It is a nice feature.